src/ui/Facet/Facet.ts

```diff
@@ -572,12 +574,14 @@ export class Facet extends Component {
      * Specifies a JSON object describing a mapping of facet values to their desired captions. See
      * [Normalizing Facet Value Captions](https://developers.coveo.com/x/jBsvAg).
      *
-     * **Note:**
-     * > You cannot set this option directly in the component markup as an HTML attribute. You must either set it in the
-     * > [`init`]{@link init} call of your search interface (see
-     * > [Components - Passing Component Options in the init Call](https://developers.coveo.com/x/PoGfAQ#Components-PassingComponentOptionsintheinitCall)),
-     * > or before the `init` call, using the `options` top-level function (see
-     * > [Components - Passing Component Options Before the init Call](https://developers.coveo.com/x/PoGfAQ#Components-PassingComponentOptionsBeforetheinitCall)).
+     * **Notes:**
+     * > * You cannot set this option directly in the component markup as an HTML attribute. You must either set it in the
+     *    > [`init`]{@link init} call of your search interface (see
+     *    > [Components - Passing Component Options in the init Call](https://developers.coveo.com/x/PoGfAQ#Components-PassingComponentOptionsintheinitCall)),
+     *    > or before the `init` call, using the `options` top-level function (see
+     *    > [Components - Passing Component Options Before the init Call](https://developers.coveo.com/x/PoGfAQ#Components-PassingComponentOptionsBeforetheinitCall)).
+     *
+     * > *  Using value captions will disable alphabetical sorts.
```

Comment:
I think it would be a good idea to add a link to the availableSorts option:

> * Using value captions will disable alphabetical sorts (see the [`availableSorts`]{@link Facet.options.availableSorts} option).